GOVERNANCE.md

```diff
@@ -16,64 +17,125 @@ Users activities include (but are not limited to):
+* Act as an advocate for the Project in events and support the project’s communication online, as time allows.
 
-In all cases silence is consent.  
+#### How to be part of the Steering Committee
+Steering Committee are invited by the Advisory Committee
 
-### Consensus Building
+#### Current List of Steering Committee Members
+* Chris Erdmann (University of North Carolina, RENCI)
+* Alicia Fagerving (Wikimedia Sverige)
+* Sandra Fauconnier (Wikimedia Foundation)
+* Rufus Pollock (Datopian, Open Knowledge Foundation)
+* Simon Rogers (Google News Initiative)
+* Lozana Rossenova (Rhizome)
+* Juliane Schneider (Harvard Catalyst, Clinical and Translational Science Center)
+* Wesley Sherperd (Unifyd Insights)
+* Fabio Tacchelli (Siren Solutions)
+* Ruben Verborgh (University of Ghent)
 
-If you feel that lazy consensus isn't appropriate for your proposal, you can explicitly request for feedback on the [developer discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine-dev). [Building consensus](http://community.apache.org/committers/consensusBuilding.html) help contributors and committers to gather feedback early on and pool the interest by the community for a new feature.
+### Advisory Committee
+The Advisory Committee runs the adminstrative aspect of the project on a day to day basis with the support of Code for Science and Society (CS&S). Its member are bound by the  fiscal sponsorship agreement with Code for Science and Society. Meet once per month with Code for Science and Society. 
 
-Everyone is invited to express their opinion of any given feature or pull request. Support is expressed using:
+It is composed of at least three members. No more than 49 percent of the members of the Advisory Committee may be paid by the projcet.
 
-- +1 means: "I agree with this and will help make it happen."
-- +0.9 means: 'This is a cool idea and I like it, but I don't have time/the skills necessary to help out.'
-- +0 means: "I agree with this but probably won't make it happen, so my opinion is not that important."
-- -0 means: "I don't agree with this, but I'm offering no alternative so my opinion is not that important."
-- -0.9 means: 'I really don't like this, but I'm not going to stand in the way if everyone else wants to go ahead with it.' 
-- -1 means: "I don't agree and I am offering an alternative that I am able to help implement."
+Advisory Committee members: 
+* Represent OpenRefine with Code for Science and Society 
+* Provide guidance and oversight of the Project’s staff and operations;
+* Approve budgets, and contracts
+* Vote to terminate contracts when necessary.
+* Participate to at least one call per quarter;
+* Help advocate for the Project;
+* Help the Project build connections and partnerships;
+* Are part of the Admin team for the project on Github
```

Comment:
Hmm, I don't think there needs to be an explicit requirement that advisory committee members MUST be part of the Admin team.  But if others disagree, that's fine. We just need to get consensus on what qualifies to be an Admin for the GitHub project or not.  Perhaps Roles and Responsibilities on GitHub can be put into a new section.  I don't know.

```suggestion
* Can be part of the Admin team for the project on GitHub
```

Reply:
Good point @thadguidry  - In that case I will remove the mention of the GitHub admin team entirely.